pom.xml

```diff
@@ -169,7 +169,7 @@
       <dependency>
          <groupId>io.openshift</groupId>
          <artifactId>booster-catalog-service</artifactId>
-         <version>3</version>
+         <version>4-SNAPSHOT</version>
```

Comment:
I'd avoid SNAPSHOTS here because they are not published or built in the CI environment